protos/farm_ng/core/image.proto

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Huh, I thought we wanted to move the whole sophus2 folder?

Reply:
not yet, this just the protobufs for now.  In a follow up we'll move the sophus2 folder.